### ISSUE TITLE
False positive: Disallow calling some global objects as functions

### ISSUE THREAD

**Issue by coderaiser** [oxc-project/oxc#1733]
![image](https://github.com/oxc-project/oxc/assets/1573141/5d9e41ca-d3f3-4112-99e4-db7aded0f6ff)

Same here:

![image](https://github.com/oxc-project/oxc/assets/1573141/f658b7cc-9b2b-4774-814a-2c5b379f1ae8)

How can I disable `no-obj-calls` rule?

This doesn't works:

```sh
oxlint -D no-obj-attern=fixture -D no-obj-calls
```

**Comment by camc314** [comment]
`-D no-obj-calls` will deny (emit an error)

try `-A no-obj-calls` instead (this will allow the rule - supress the lint)


For no-useless-spread


> Same here:
> 
> ![image](https://private-user-images.githubusercontent.com/1573141/291292554-f658b7cc-9b2b-4774-814a-2c5b379f1ae8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI5MDg1MzIsIm5iZiI6MTcwMjkwODIzMiwicGF0aCI6Ii8xNTczMTQxLzI5MTI5MjU1NC1mNjU4YjdjYy05YjJiLTQ3NzQtODE0YS0yYzViMzc5ZjFhZTgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIxOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMThUMTQwMzUyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjVhNGU2ODA5ODQ4Yjk2YTRiMDFlNDgxODUxZmQzOGUzOGI5MjgyOTEwZGVkM2JmNGQzZTRkNDUwYmY5MWE0ZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.tePO5rt8GMV0OMEguVgLvtRpeRAfWPpQzojtT4PH7Tw)

I think this lint looks correct, by doing `...[]`, a new array is being created and then unneccessarily spread. 
Changing it to
```ts
expression.openingElement.attributes.unshift(nameAttribute, onChangeAttribute)
```

should fix the error
